filters.py

```diff
@@ -0,0 +1,154 @@
+        self._db = db
+        self._client = client
+        self._me = await client.get_me()
+
+    async def filtercmd(self, message):
+        """Adds a filter into the list."""
+        args = (utils.get_args_raw(message)).strip()
+        reply = await message.get_reply_message()
+        chatid = str(message.chat_id)
+        filters = self._db.get("FilterModule", "filters")
+        if not args:
+            await message.edit("<b>Enter a name for the filter first!</b>")
+            return
+        else:
+            args = args.split(" ")
+            name = args[0]
+        if not filters:
+            filters = {}
+            filters.setdefault(chatid, {})
+        if chatid not in filters:
+            filters.setdefault(chatid, {})
+        if not reply:
+            if len(args) == 1:
+                await message.edit("<b>Please reply to a message or enter a text to save as filter.!</b>")
+                return
+            else:
+                value = " ".join(args[1::])
+                msg_to_log = await self._db.store_asset(value)
+        else:
+            value = reply.text
+            msg_to_log = await self._db.store_asset(reply)
+        filters[chatid][name] = str(msg_to_log)
+        self._db.set("FilterModule", "filters", filters)
+        await message.edit("<b>Text ''{}'' has been successfully filtered in this chat.</b>".format(name))
+
+    async def stopcmd(self, message):
+        """Removes a filter from the list."""
+        filtern = str(utils.get_args_raw(message))
+        filters = self._db.get("FilterModule", "filters")
+        chatid = str(message.chat_id)
+        if not filtern:
+            await message.edit("<b>Please specify the name of the filter.</b>")
+            return
+        try:
+            del filters[chatid][filtern]
+            await message.edit("<b>Filter ''{}'' successfully removed from the chat.</b>".format(filtern))
+            self._db.set("FilterModule", "filters", filters)
+        except Exception:
+            await message.edit("<b>Filter ''{}'' not found in this chat</b>".format(filtern))
+
+    async def stopallcmd(self, message):
+        """Clears out the filter list."""
+        filters = self._db.get("FilterModule", "filters")
+        chatid = str(message.chat_id)
+        try:
+            del filters[chatid]
+            self._db.set("FilterModule", "filters", filters)
+            await message.edit("<b>All filters successfully removed from the chat.</b>")
+        except Exception:
+            await message.edit("<b>There are no filters to clear out in this chat.</b>")
+
+    async def filterscmd(self, message):
+        """Shows saved filters."""
+        filters = ""
+        filt = self._db.get("FilterModule", "filters")
+        chatid = str(message.chat_id)
+        try:
+            for i in filt[chatid]:
+                filters += "<b> -  " + str(i) + "</b>\n"
+                pass
+        except Exception:
+            pass
```

Comment:
No!!!!!!!!!!!!!!!!!!!! Just make the exception impossible, by setting a default value